rrfs-test/IODA/yaml/prepbufr_aircar.yaml

```diff
@@ -0,0 +1,361 @@
+# (C) Copyright 2024 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+      obsdatain: "./bufr/2024052700.rap.t00z.prepbufr.tm00"
+
+      exports:
+        #group_by: prepbufrDataLvlCat
+        subsets:
+          - AIRCAR    # MDCRS ACARS AIRCRAFT REPORTS (ACID)
+          #- AIRCFT    # AIREP, PIREP, AMDAR, TAMDAR AIRCRAFT REPORTS (ACID)
+        variables:
+          # MetaData
+          timestamp:
+            timeoffset:
+              timeOffset: "*/DHR"
+              transforms:
+                - scale: 3600
+              referenceTime: "2024-05-27T00:00:00Z"
+          timeOffset:
+            query: "*/DHR"
+          longitude:
+            query: "*/XOB"
+          latitude:
+            query: "*/YOB"
+          stationIdentification:
+            query: "*/SID"
+          stationElevation:
+            query: "*/ELV"
+          prepbufrReportType:
+            query: "*/TYP"
+          dumpReportType:
+            query: "*/T29"
+          prepbufrDataLvlCat:
+            query: "*/PRSLEVLA{1}/CAT"
+          aircraftPhase:
+            query: "*/PRSLEVLA{1}/ACFT_SEQ/POAF"
+          aircraftFlightNumber:
```

Comment:
The aircraft flight number is different from the aircraft tail number.
An aircraft's flight number identifies a specific flight, while its tail number identifies the aircraft itself

Do we know what ACID refers to here? Maybe quickly dumping a few example data will give us a good idea. Thanks!

Reply:
Good to know. I was not aware the tail number is in `SID`. Thanks, @delippi